src/features/AuthModal/AuthModal.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { componentRender } from 'shared/lib/tests';
+import { AuthModal } from './AuthModal';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('AuthModal', () => {
+    const renderComponent = () => componentRender(
+        <AuthModal open onClose={() => {}} />,
+        {},
+    );
+
+    test('should renders without error', () => {
+        renderComponent();
+
+        const wrapper = screen.getByTestId('AuthModal-wrapper');
+        const title = screen.getByTestId('AuthModal-title');
+        const switchBtn = screen.getByTestId('AuthModal-caption-switch');
+        const emailField = screen.getByTestId('AuthModal-email');
+        const passField = screen.getByTestId('AuthModal-pass');
```

Comment:
you can keep it as is if you preffer but I'm not realy sure what's the point to store them as a separate const

Reply:
Fixed